packages/tools/pie-icons-webc/playwright/index.html

```diff
@@ -52,6 +52,5 @@
 </head>
 <body>
     <div id="root"></div>
-    <script type="module" src="./index.ts"></script>
```

Comment:
Should this file also have been deleted?

Reply:
Yeah you're right. I was hoping this could be deleted as it's just an empty file, but I looks like playwright probably uses this file to inject the component code.